Makefile

```diff
@@ -7,16 +7,15 @@ uninstall: FORCE
 	pip uninstall scvi-distributed
 
 lint: FORCE
-	flake8
+	ruff check .
```

Comment:
I would recommend sticking to the same linting rules as CAS has. Which basically the same thing as you have, but the line length is 120. You can find the example here:
https://github.com/cellarium-ai/cellarium-cloud/blob/main/tox.ini#L47

We can review and discuss the preferred line length and may change it in CAS sometime later, but I think we should be consistent on this thing. Also, the default value in black is 88 which seems a little bit too short these days especially having the type annotations.